rollup.config.ts

```diff
@@ -144,4 +146,24 @@ const buildNative = {
   ],
 }
 
-export default [buildNode, buildNative, buildEsm, buildUdm]
+/**
+ * Configuration for the iife build
+ */
+const buildIife = {
+  input: 'src/index.ts',
+  output: {
+    file: 'lib/iife/index.js',
```

Comment:
Should we specify this entry in `package.json`? I'm afraid I don't know how IIFE modules are being resolved. Is it sufficient to explicitly provide the path to the module when importing?

Reply:
There is a `browser` field in package.json that can be used for just that, however that causes an issue with webpack. When webpack finds both a module and a browser field, it will choose the browser field, which in this case is the less desirable iife format which doesn't allow tree shaking. So in this case, since we want webpack to use the esm format under the module field, we should not specify a value for browser.

The way I saw this being used is by either manually copying the iife to whatever app the users are using, or by linking directly to the file using a cdn like is explained here: https://mswjs.io/docs/recipes/usage-with-cdn but on the iife. I am happy contributing that documentation should this change be accepted.

So not having the browser field specified in package.json makes it a little less obvious that an iife format is available, but hopefully good documentation can somewhat fill that gap.

Reply:
Thank you for such an insightful explanation. That documentation page was also something that came to my mind when I saw this pull request. I think starting with the documentation is always a good choice, and we'd be happy to welcome your pull request to extend that page once this lands.